devenv/dev-dashboards/dashboards_test.go

```diff
@@ -0,0 +1,84 @@
+package dev_dashboards
+
+import (
+	"encoding/json"
+	"io"
+	"io/fs"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"cuelang.org/go/cue/errors"
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/grafana/pkg/coremodel/dashboard"
+	"github.com/grafana/grafana/pkg/cuectx"
+)
+
+func TestDevenvDashboardValidity(t *testing.T) {
```

Comment:
We spent a long time last year figuring out where these devenv tests should live. The quagmire of stuff under `pkg/schema` kinda required we work in there, and also felt like the least-bad place to put it.

Now, though, it's much cleaner and clearer to have these tests in devenv itself, and then just depend on the dashboard coremodel, much like how we expect other code should. (Those old tests will go away, soon enough...)